bindings/python/opendal.pyi

```diff
@@ -18,6 +18,7 @@ class Error(Exception): ...
 class Operator:
     def __init__(self, scheme: str, **kwargs): ...
     def read(self, path: str) -> bytes: ...
+    def reader(self, path: str) -> Reader: ...
```

Comment:
How about rename it to `open_reader`?

Reply:
I prefer rename to `open` and return an enum which could be `Reader` or `Writer`. But add `open_reader` first still a good choice.